src/modules/EventTypes.js

```diff
@@ -11387,7 +11387,75 @@ export const EventTypes = {
               {
                 label: 'Full Hostname',
                 name: 'fullHostname'
-              }
+              },
+              {
+                label: 'Cluster Name',
+                name: 'clusterName'
+              },
+              {
+                label: 'Core Count',
+                name: 'coreCount'
+              },
+              {
+                label: 'CPU IO Wait Percent',
+                name: 'cpuIOWaitPercent'
+              },
+              {
+                label: 'Display Name',
+                name: 'displayName'
+              },
+              {
+                label: 'Entity GUID',
+                name: 'entityGuid'
+              },
+              {
+                label: 'Entity Key',
+                name: 'entityKey'
+              },
+              {
+                label: 'Instance Type',
+                name: 'instanceType'
+              },
+              {
+                label: 'NR Entity Type',
+                name: 'nr.entityType'
+              },
```

Comment:
Do we want to expose the NR attributes here?

Reply:
These attributes come in on the 1st party and accounts got points for custom attributes where there were no custom attributes as these were included, so as this is shared between 3rd party and 1st party I've added them here. An alternative could be to override and augment this in 1st party.